src/cli/cli_srp_server.cpp

```diff
@@ -43,6 +43,26 @@
 namespace ot {
 namespace Cli {
 
+/**
+ * @cli srp server addrmode (set,get)
```

Comment:
Switch to be (get,set)

I just realized you did the client ones like this too.  They should've all been (get,set) for consistency with all the other commands we've already documented.  Can you make that update in this PR too?